CMakeLists.txt

```diff
@@ -135,7 +135,7 @@ endif()
 # shared WarpX library or a third party, like ImpactX, uses ablastr in a
 # shared library (e.g., for Python bindings), then we need relocatable code.
 option(ABLASTR_POSITION_INDEPENDENT_CODE
-    "Build ABLASTR with position independent code" ${WarpX_LIB})
+       "Build ABLASTR with position independent code" ON)
```

Comment:
We can default to ON here, which is the reasonable thing to do to use ABLASTR in both shared and static libs downstream.